extensions_admin/pulp_rpm/extensions/admin/upload/environment.py

```diff
@@ -0,0 +1,70 @@
+from gettext import gettext as _
+
+from pulp.client.commands.options import OPTION_REPO_ID
+from pulp.client.commands.repo.upload import UploadCommand
+from pulp.client.extensions.extensions import PulpCliOption
+
+from pulp_rpm.common.ids import TYPE_ID_PKG_ENVIRONMENT
+
+
+NAME = 'environment'
+DESC = _('creates a new package environment')
+
+d = _('id of the package environment')
+OPT_ENV_ID = PulpCliOption('--environment-id', d, aliases=['-i'], required=True)
+
+d = _('name of the package environment')
+OPT_NAME = PulpCliOption('--name', d, aliases=['-n'], required=True)
+
+d = _('description of the package environment')
+OPT_DESCRIPTION = PulpCliOption('--description', d, aliases=['-d'], required=True)
```

Comment:
Should the description be required? A lot of Pulp things don't require the description.